### ISSUE TITLE
Chat Not working 

### ISSUE THREAD

**Issue by sharemyguitar** [SocialEngine/phpv4-issues#6]
I updated to the most recent version of SE and the most recent version of PHP. I then changed the Chat plugin option from 'Everyone' to 'Friends Only' in the SE admin area and noticed that it was always displaying (0) friends online even when they were online. So I changed it back to 'Everyone' and it remains always on (0) friends online.

**Comment by shivani2910** [comment]
Hi @sharemyguitar. Such issue usually happens when the task scheduler isn't running frequently enough because of which "Chat Data Maintenance" task also does not run properly and online friends / users are not updated timely . You may check the Task Scheduler on your website ( https://example.com/admin/tasks ) and check the last time when "Chat Data Maintenance" had run successfully.

If this has not run since a long time, then we would suggest you to do below changes in the "Task Scheduler Settings" section ( https://example.com/admin/tasks/settings ):

1) Change the "Trigger Method" to "Cron". Please make sure that before switching to cron, you get it set-up on your server by your hosting provider.

2) Change the value of "Concurrent Processes" from 2 to 4 and "Tasks Run per Request" from 2 to 1 . This will also help your task manager run more frequently.

After doing these changes, If the issue still persists on your website, please contact our support team and they'll help you.

Best Regards,
Shivani

**Comment by sharemyguitar** [comment]
Hi Shivani,

Thank you for the reply back. I just checked the "Chat Data Maintenance" as you suggested and it seems to be running properly and often without failing. See the screen shot below. 
![2-18-2015 7-53-56 am](https://cloud.githubusercontent.com/assets/11047540/6250547/6adbc586-b743-11e4-9119-0b2f2042adca.png)

**Comment by shivani2910** [comment]
Hi Mickey,

Thanks for your response. If Task Scheduler is not an issue and problem has occurred after you got the latest version of PHP updated on your server then please check doing below changes in ```"/application/index.php"``` file :

Please search for below code:
```
error_reporting(E_ALL);
```
and replace the above code with below one:
```
error_reporting(E_ALL & ~E_STRICT & ~E_DEPRECATED);
```
If the issue still persists after doing the above change then please open a support ticket along with the details of your website to enable us look into this issue in your SocialEngine installation.

We appreciate your support and cooperation.

Best Regards,
Shivani

**Comment by sharemyguitar** [comment]
Hi Shivani,

Thanks for the suggestion. I made the change, cleared cache and the problem persists, I have some other issues as well from the recent updates so I guess I am going to need to open a support ticket. 

Thanks again,
Mickey

**Comment by shivani2910** [comment]
Hi Mickey,

I've seen your comment on our blog post ( http://blog.socialengine.com/2015/02/03/socialengine-php-4-8-7-released/#comment-1865456973 ) mentioning that the issue has been resolved at your end by downgrading the PHP version. As I said that we''ll be investigating it further and once the problem is spotted by our team, we'll resolve it in next version.

Thanks for your feedback.

Best Regards,
Shivani

**Comment by smazurov** [comment]
we've just released 4.8.8 that fixed some issues with newer php versions, could you upgrade and test again?

**Comment by sharemyguitar** [comment]
I had downgraded PHP in order to fix the chat issue. Not sure I want to upgrade to the latest version and find that it causes issues again now that I have everything working. What benefit would I have to upgrade to the latest version of PHP?

**Comment by smazurov** [comment]
no benefit for SE PHP, so unless you need/want to run latest version of PHP, the only benefit would be verification that this was fixed. 

We'll close the issue for now and I'm sure we'll have somebody else chime in if the fix in 4.8.8 doesn't help.
